sdk/servicebus/azure-servicebus/migration_guide.md

```diff
@@ -1,164 +1,444 @@
+- using the new [`azure-identity`](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/README.md) library
+to share a single authentication approach between clients
+- a unified logging and diagnostics pipeline offering a common view of the activities across each of the client libraries
+
+## Important changes
+
+### Client hierarchy
+
+In the interest of simplifying the API surface we've made a single top level client called `ServiceBusClient`,
+rather than one for each of queue, topic, and subscription.
+This acts as the single entry point in contrast with multiple entry points from before.
+You can create senders and receivers from this client to the queue/topic/subscription/session of your
+choice and start sending/receiving messages.
+
+#### Approachability
+By having a single entry point, the `ServiceBusClient` helps with the discoverability of the API
+as you can explore all available features through methods from a single client, as opposed to searching
+through documentation or exploring namespace for the types that you can instantiate.
+Whether sending or receiving, using sessions or not, you will start your applications by constructing the same client.
+
+#### Consistency
+We now have methods with similar names, signature and location to create senders and receivers.
+This provides consistency and predictability on the various features of the library.
 
-This document is intended for users that are familiar with v0.50 of the Python SDK for Service Bus library (`azure-servicebus 0.50.x`) and wish 
-to migrate their application to v7 of the same library.
+### Client constructors
 
-For users new to the Python SDK for Service Bus, please see the [readme file for the azure-servicebus](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/README.md).
+- While we continue to support connection strings when constructing a client, the main difference is when using Azure Active Directory.
+We now use the new [`azure-identity`](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/identity/azure-identity/README.md) library
+to share a single authentication solution between clients of different Azure services.
+- `QueueClient`, `TopicClient` and `SubscriptionClient` have been removed, to create senders and receivers, please check [Creating sender and receivers](#creating-sender-and-receivers)
 
-## General changes
-Version 7 of the azure-servicebus package is the result of our efforts to create a client library that is user-friendly and idiomatic to the Python ecosystem.
-Alongside an API redesign driven by the new [Azure SDK Design Guidelines for Python](https://azure.github.io/azure-sdk/python_introduction.html#design-principles), 
-the latest version improves on several areas from v0.50.
 
-Note: The large version gap is in order to normalize service bus SDK versions across our languages, as they consolidate on structure as well.
+In V0.50:
```

Comment:
What is the reason the table format is changed? Also, I think there's a great value in providing the relevant sample in place